README.md

```diff
@@ -99,9 +101,9 @@ After you're finished please remove all the comments and instructions!
 
 > Describe between 1-3 key features of the application.
```

Comment:
- [ ] Please remove this placeholder texts from your Readme, they are there to help us fill the appropriate information in the readme file and not be present in our final outcome, please remove all such placeholders from your Readme